packages/app/src/features/spark-rewards/views/RewardsView.tsx

```diff
@@ -14,7 +14,7 @@ export function RewardsView({ ongoingCampaignsResult, activeRewardsResult }: Rew
   return (
     <PageLayout>
       <h1 className="typography-heading-1">Rewards</h1>
-      <div className="grid grid-cols-1 gap-5 md:grid-cols-[11fr_5fr]">
+      <div className="grid grid-cols-1 gap-5 lg:grid-cols-[11fr_5fr]">
```

Comment:
Maybe even 
```suggestion
      <div className="grid grid-cols-1 gap-5 xl:grid-cols-[11fr_5fr]">
```
?
https://www.chromatic.com/test?appId=6629151a3bd0c7efaaab1e03&id=67bdd20844c0998e62c969b1